README.md

```diff
@@ -48,14 +48,16 @@ stackdriver-prometheus-sidecar --help
 
 #### Filters
 
-The `--filter` flag allows to provide filters which all series have to pass before being sent to Stackdriver. The flag may be repeated to provide several filters. Filters use the same syntax as the well-known PromQL label matchers, e.g.:
+The `--filterset` flag allows to provide filters which all series have to pass before being sent to Stackdriver. Filters use the same syntax as the well-known PromQL label matchers, e.g.:
```

Comment:
Thoughts:

1. If using promQL for metric selector, it allows more selectors than label matcher only, e.g. querying by metric name. Add more example and provide a point to [Prometheus' time-series-selector document]( https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors) helps. Prometheus API also [uses](https://github.com/prometheus/prometheus/blob/8c8bb82d04e854d82555150ca2e71e2bc159df90/web/api/v1/api.go#L478
) the `ParseMetricSelector` you used, so the Prometheus API should be a good reference on what's allowed.

3. If using promQL style for the label matcher, I think adding curly bracket for label-matcher matches what Prometheus users experience.
which means: `__name__!~"cadvisor_.+",job="k8s"` -> `{__name__!~"cadvisor_.+",job="k8s"}`

Reply:
Thanks! I've fixed the brackets, added a link to Prometheus docs, and included another example that uses metric name matching with a simpler format.